.buildbot.sh

```diff
@@ -6,11 +6,10 @@ export CARGO_HOME="`pwd`/.cargo"
 export RUSTUP_HOME="`pwd`/.rustup"
 
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > rustup.sh
-# Install no toolchain initially to ensure toolchain rollback.
-sh rustup.sh --default-host x86_64-unknown-linux-gnu --default-toolchain none -y --no-modify-path
-
+sh rustup.sh --default-host x86_64-unknown-linux-gnu --default-toolchain nightly -y --no-modify-path
```

Comment:
can we select the minimal profile here? If so we can kill the following invocation of rustup?

Reply:
We can. Fixed in https://github.com/softdevteam/rboehm/pull/19/commits/0a4753601def7808deb3a6292c56f1e807eee195.